### ISSUE TITLE
mob_init: ImportError: No module named ete3

### ISSUE THREAD

**Issue by tseemann** [phac-nml/mob-suite#34]
Just noticed this error  in the _middle/end_ of the `mob_init` process?

```
2019-11-05 12:30:36,543 INFO: Sketching complete plasmid database [in /home/linuxbrew/.linuxbrew/opt/python/lib/python3.7/site-packages/mob_suite/mob_init.py:215]
2019-11-05 12:30:52,922 INFO: Init ete3 library ... [in /home/linuxbrew/.linuxbrew/opt/python/lib/python3.7/site-packages/mob_suite/mob_init.py:227]
Traceback (most recent call last):
  File "<string>", line 1, in <module>
ImportError: No module named ete3
2019-11-05 12:30:52,960 INFO: MOB init completed successfully [in /home/linuxbrew/.linuxbrew/opt/python/lib/python3.7/site-packages/mob_suite/mob_init.py:241]
```

I have ete3:
```
pip3up ete3
Requirement already up-to-date: ete3 in ./.linuxbrew/lib/python3.7/site-packages (3.1.1)
```

**Comment by kbessonov1984** [comment]
What does python -c "import ete3" command return? It looks like ete3 library did not install correctly even though pip reports it as being installed (ete3 has so many dependencies ...). In the mob_init it should start downloading the taxonomy file from NCBI that will help with taxid conversion to full taxonomic lineage. I did not see this issue when testing PyPI mob_suite 2.0.1 package. I will test it tomorrow too. ete3 is so picky ... that is why we've removed tree PNG rendering as so many users had issues with mesa drivers on their clusters. I have a strong desire to rewrite ete3 dependencies to make mob_suite more system agnostic.

**Comment by kbessonov1984** [comment]
I've just tried to run pip install on our HPC cluster in Winnipeg (waffles) and pip installed ete3 there correctly. I use a clean conda environment in order not to interfere with the system libraries. All dependencies were installed via pip (except to scipy which is a bit temperamental). I recommend installing ete3 via conda if pip does not work.

```
% (mobsuitepip) [kbessono@waffles ~]$ python -c "import ete3"
Traceback (most recent call last):
  File "<string>", line 1, in <module>
ModuleNotFoundError: No module named 'ete3'

#Let's install this missing package via pip (not conda)
% pip install ete3
Building wheels for collected packages: ete3
  Building wheel for ete3 (setup.py) ... done
  Created wheel for ete3: filename=ete3-3.1.1-cp37-none-any.whl size=2226264 sha256=8afbc2d10976d58cf26fe1e25ab5141469e76dd32470bbd37d9dd5753e67906c
  Stored in directory: /home/CSCScience.ca/kbessono/.cache/pip/wheels/14/90/50/5c10d965e7fb68343d9c5349cce2787ff3e38397946468d640
Successfully built ete3
Installing collected packages: ete3, PyQt5-sip, pyqt5
Successfully installed PyQt5-sip-12.7.0 ete3-3.1.1 pyqt5-5.13.2

#Let's see if we can import it without any errors
(mobsuitepip) [kbessono@waffles ~]$ python -c "import ete3"

#Let's install mob_suite 2.0.1
% pip install mob_sutite==2.0.1
Building wheels for collected packages: mob-suite
  Building wheel for mob-suite (setup.py) ... done
  Created wheel for mob-suite: filename=mob_suite-2.0.1-cp37-none-any.whl size=31457968 sha256=5a52f0ab92302debf0ff4d7df303f38d47d3e7074c60ec67062f1375b25c03a5
  Stored in directory: /home/CSCScience.ca/kbessono/.cache/pip/wheels/71/09/77/7bbf6b90a4778bae6f2ac303e687f3cff3b8b5d698f2f403bf
Successfully built mob-suite
Installing collected packages: mob-suite
  Found existing installation: mob-suite 2.0.0
    Uninstalling mob-suite-2.0.0:
      Successfully uninstalled mob-suite-2.0.0
Successfully installed mob-suite-2.0.1

#Let's run mob_init
%mob_init
<...>
2019-11-04 21:05:12,797 INFO: Init ete3 library ... [in /home/CSCScience.ca/kbessono/.conda/envs/mobsuitepip/lib/python3.7/site-packages/mob_suite/mob_init.py:227]
Downloading taxdump.tar.gz from NCBI FTP site (via HTTP)...
Done. Parsing...
Loading node names...
2179768 names loaded.
212398 synonyms loaded.
Loading nodes...
2179768 nodes loaded.
Linking nodes...
Tree is loaded.
Updating database: /home/CSCScience.ca/kbessono/.etetoolkit/taxa.sqlite ...
 2179000 generating entries... 
Uploading to /home/CSCScience.ca/kbessono/.etetoolkit/taxa.sqlite

Inserting synonyms:      210000 
Inserting taxid merges:  55000 
Inserting taxids:       2175000 
2019-11-04 21:08:25,549 INFO: MOB init completed successfully [in /home/CSCScience.ca/kbessono/.conda/envs/mobsuitepip/lib/python3.7/site-packages/mob_suite/mob_init.py:241]
```

**Comment by tseemann** [comment]
I think the problem is somehow connected to this 3 year old bug:
https://github.com/etetoolkit/ete/issues/195

I'll tell the postdocs to use it on conda elsewhere.

**Comment by kbessonov1984** [comment]
I was doing more testing on Debian image and also found that ete3 installs with no errors, but when importing NCBITaxa submodule from ete3, it fails due to dependency on package six. We had included this requirement in `setup.py` and re-released pip package with version 2.0.1.0 (as PyPI does not allow to replace existing versions). I agree that conda is the easiest way to install mob_suite as it depends on circlator, amos, mash and an array of python libraries. We hope that this pip would also work.

**Comment by tseemann** [comment]
FYI - `pip` has issues with 4 digit versions X.Y.Z.W. Stick to semantic versioning https://semver.org
